### ISSUE TITLE
Doesn't report error status in `execute_reply` msg

### ISSUE THREAD

**Issue by jayendra13** [Calysto/matlab_kernel#113]
Version info
```
matlab        : 9.5.0.944444 (R2018b)
matlab_kernel : 0.15.1
```
I am running a matlab [code](https://gist.github.com/jayendra13/b9e1fb438b86e68a21fafeca9d8c505c) with `matlab_kernel` having a syntax error, and tracking the messages from the _iopub_ channel.
This is the `execute_reply` msg from the _iopub_ channel.

```
{'buffers': [],
 'content': {u'execution_count': 1,
             u'payload': [],
             u'status': u'ok',
             u'user_expressions': {}},
 'header': {u'date': datetime.datetime(2018, 10, 4, 6, 11, 22, 642374, tzinfo=tzutc()),
            u'msg_id': u'1f698f2d-0d99bf43400c21a4a318708d',
            u'msg_type': u'execute_reply',
            u'session': u'94bae730-e12ee227be016e5e947bf704',
            u'username': u'jayendra',
            u'version': u'5.3'},
 'metadata': {u'started': u'2018-10-04T06:11:22.406073Z'},
 'msg_id': u'1f698f2d-0d99bf43400c21a4a318708d',
 'msg_type': u'execute_reply',
 'parent_header': {u'date': datetime.datetime(2018, 10, 4, 6, 11, 16, 937573, tzinfo=tzutc()),
                   u'msg_id': u'e5c2386c-c924e5654ae5e26a59336730',
                   u'msg_type': u'execute_request',
                   u'session': u'3851d47e-5fda30d441f1814fdb335ba5',
                   u'username': u'jayendra',
                   u'version': u'5.3'}}

```
When matlab encounter the error while executing the code, the `status` (under the `content `) should be `error` instead of `ok`.

ref: https://jupyter-client.readthedocs.io/en/stable/messaging.html#request-reply

**Comment by blink1073** [comment]
Thanks for the report, @jayendra13!  I suspect this problem exists in `metakernel`, which the `matlab_kernel` is built upon.  I don't have the cycles currently to dig into it...

**Comment by jayendra13** [comment]
Thanks @blink1073 for the prompt response.
This thing is critical to my project, should I re-raise this in `metakernel`  repo ?
or can you give any pointers where should I look for this ?

**Comment by blink1073** [comment]
I suspect it is [here](https://github.com/Calysto/metakernel/blob/0d5dafeec49e3cf68d5d9ab63bff6cf98a7d3664/metakernel/process_metakernel.py#L56).  Yes, please, I think the issue is better suited on that repo.

**Comment by jayendra13** [comment]
moved here
https://github.com/Calysto/metakernel/issues/175

**Comment by jayendra13** [comment]
@blink1073 I think this could be issue a issue https://github.com/Calysto/metakernel/issues/175#issuecomment-427311115

**Comment by jayendra13** [comment]
@blink1073 I have made some fixes for this, how can submit a PR ?

**Comment by blink1073** [comment]
Are the fixes here, in metakernel, or both?

**Comment by jayendra13** [comment]
only here.

**Comment by jayendra13** [comment]
or you can give me access to push a new branch here ?

**Comment by blink1073** [comment]
1) Fork the repo
2) Create a new branch for the changes
3) commit your changes to the branch
4) push the branch to your <origin>
5) Open a PR on github
6) I'll review and merge

**Comment by blink1073** [comment]
This site looks pretty good for an overview: https://services.github.com/on-demand/github-cli/open-pull-request-github

**Comment by jayendra13** [comment]
Done please review that.
Thanks

**Comment by blink1073** [comment]
Fixed by #114.
